state-chain/runtime/src/chainflip/address_derivation/btc.rs

```diff
@@ -49,7 +49,7 @@ impl AddressDerivationApi<Bitcoin> for AddressDerivation {
 /// Derives the BTC vault deposit address from the private channel id.
 /// Note: This function will **panic** if the private channel id is out of bounds or if there is
 /// no active epoch key for Bitcoin.
-pub(crate) fn derive_btc_vault_deposit_address(private_channel_id: u64) -> String {
+pub fn derive_btc_vault_deposit_address(private_channel_id: u64) -> String {
```

Comment:
We might have to alter this method to allow us to generate the previous key too. The deposit monitor will need to track this.